lambdas/evaluate-gpg45-scores/src/main/java/uk/gov/di/ipv/core/evaluategpg45scores/EvaluateGpg45ScoresHandler.java

```diff
@@ -174,21 +173,26 @@ private Map<String, Object> buildJourneyErrorResponse(ErrorResponse errorRespons
     }
 
     @Tracing
-    private boolean hasMatchingGpg45Profile(
+    public boolean hasMatchingGpg45Profile(
```

Comment:
Has this been made public so it can be tested - can it not be tested indirectly through its caller?

Reply:
Yep, I've just made it private again. I wanted to have more obvious testing but Dan made the same comment earlier